core/common/src/rpc/IModelReadRpcInterface.ts

```diff
@@ -59,31 +61,39 @@ export abstract class IModelReadRpcInterface extends RpcInterface {
     NOTE: Any add/remove/change to the methods below requires an update of the interface version.
     NOTE: Please consult the README in this folder for the semantic versioning rules.
   ===========================================================================================*/
+  @RpcOperation.allowResponseCaching(RpcResponseCacheControl.Immutable)
   public async getConnectionProps(_iModelToken: IModelRpcOpenProps): Promise<IModelConnectionProps> { return this.forward(arguments); }
```

Comment:
I think this should warrant a bump in the `interfaceVersion` given the change. Will this not allow POST requests to be made anymore or will the backend still be able to handle both?

Reply:
The backend should be able to handle both types of requests. Although I do see a potential issue, if the frontend is newer than the backend then getConnectionProps may be turned into a GET by the frontend, but backend will still have the old cache headers which will cause problems especially for getConnectionProps. Is this scenario possible? InterfaceVersion bump will mitigate that anyways I assume. I will make the change

Reply:
i bumped the minor version